### PR TITLE
Updated visualization documentation

### PR DESCRIPTION
**Pull request by dtm2451**
visualization.R:
- Updated plotCellVsReference argument descriptions to be more specific or to match with documentation in SingleR
- Added a few extra tweaks in the examples code in plotScoreVsReference

test-visualizations.R
- moved a test that was in the wrong spot

### CODE REVIEW

File: R/visualizations.R [hunk 1]
```diff
@@ -2,19 +2,16 @@
...
 #' 
-#' @param test Numeric matrix of single-cell expression values (usually log-transformed
-#' or otherwise variance-stabilized), where rows are genes and columns are cells.
+#' @param test A numeric matrix of single-cell expression values where rows are genes and columns are cells.
 #' Alternatively, a \linkS4class{SummarizedExperiment} object containing such a matrix.
-#' @param test.id Integer scalar specifying the index of the target cell to use.
-#' Alternatively, a string containing the name of the cell.
-#' @param ref Numeric matrix of reference dataset expression values (usually log-transformed
-#' or otherwise variance-stabilized), where rows are genes and columns are cells.
+#' @param test.id Integer scalar or character specifying the index or name of the target cell to use.
```
Comment (LTLA) [inline R/visualizations.R]: "or string". It's rather unfortunate that R calls them character vectors, they should really be referred to as string vectors. If you're coming from C, character vectors suggest character arrays, which are quite different.

File: R/visualizations.R [hunk 2]
```diff
@@ -111,6 +108,16 @@ plotCellVsReference <- function(test, test.id, ref, ref.id, assay.type.test = 'l
 #' # Creating a heatmap with clusters.
 #' plotScoreHeatmap(pred, clusters=test$label)
 #' 
+#' #To only show certain labels, you can use labels.use or max.labels
```
Comment (LTLA) [inline R/visualizations.R]: Space between hash and first letter, for readability.

**Comment by LTLA** [comment]
Incidentally, double-check your `git config --global user.name` and `user.email` so that it uses your github user name and email, otherwise your commits don't get registered to your account (which makes it harder to use GitHub's interface to track down who is responsible for what).

File: R/visualizations.R (hunk 1 above)
Comment (dtm2451) [reply R/visualizations.R]: I have changed all mention of "character" to "string".  Check?

**Comment by LTLA** [comment]
FYI, if you change the docs in the `#'` sections, make sure you `devtools::document()` every now and then to update the actual man files.

File: R/visualizations.R (hunk 2 above)
Comment (dtm2451) [reply R/visualizations.R]: Added spaces after all 3 # without spaces after them.

**Pull request merged**